docs/add-ons/vault.md

```diff
@@ -0,0 +1,44 @@
+# HashiCorp Vault
+
+[HashiCorp Vault](https://www.vaultproject.io) brokers and deeply integrates with trusted identities to automate access to secrets, data, and systems.
+
+This add-on is implemented as an external add-on. For detailed documentation and usage of the add-on please refer to the add-on [repository](https://github.com/hashicorp/hashicorp-vault-eks-blueprints-addon).
```

Comment:
Getting a 404 for this link?

Reply:
You're correct @kcoleman731, this was intentional at the time.